garage/tf/algos/npo.py

```diff
@@ -50,31 +69,39 @@ def __init__(self,
                  optimizer=None,
                  optimizer_args=None,
                  name='NPO',
-                 policy=None,
                  policy_ent_coeff=0.0,
                  use_softplus_entropy=False,
                  use_neg_logli_entropy=False,
                  stop_entropy_gradient=False,
+                 entropy_method='regularized',
                  **kwargs):
         self.name = name
         self._name_scope = tf.name_scope(self.name)
         self._use_softplus_entropy = use_softplus_entropy
         self._use_neg_logli_entropy = use_neg_logli_entropy
         self._stop_entropy_gradient = stop_entropy_gradient
-
         self._pg_loss = pg_loss
         if optimizer is None:
             if optimizer_args is None:
                 optimizer_args = dict()
             optimizer = LbfgsOptimizer
 
+        if entropy_method == 'max':
+            self._maximum_entropy = True
+            self._entropy_regularzied = False
+        elif entropy_method == 'regularized':
+            self._maximum_entropy = False
+            self._entropy_regularzied = True
+        else:
+            raise NotImplementedError('Unknown entropy_method')
```

Comment:
this is a ValueError

Reply:
there is no third way to configure the entropy bonus?